proteus/Domain.py

```diff
@@ -692,67 +723,56 @@ def __init__(self,fileprefix):
         D_base.__init__(self,3,name=fileprefix)
         self.meshfile=fileprefix
 
+
 class PiecewiseLinearComplexDomain(D_base):
     """
     3D domains desribed by closed surfaces made up of general polygonal facets.
     """
-    def __init__(self,fileprefix=None,vertices=None,facets=None,
-                 facetHoles=None,holes=None,regions=None,vertexFlags=None,facetFlags=None,regionFlags=None,regionConstraints=None,
-                 name="PLCDomain",units="m"):
+    def __init__(self, fileprefix=None, vertices=None, segments=None, facets=None,
```

Comment:
There are a couple of different options. I believe tetgen allows "singular" facets, that is facets that are just edges or points, so you could actually use the segments that are provided by turning them into facets. I think you'd need to do the  same with the segment flags. The quickest  solution is probably just to remove them or raise  a loud  warning that the segments are not  currently being used. I wouldn't want somebody to go to a lot of trouble defining segments and then go through a long debugging process to figure out whey their segments don't show up in the mesh.